products/bigquery/api.yaml

```diff
@@ -383,6 +383,610 @@ objects:
+      - !ruby/object:Api::Type::NestedObject
+        name: 'configuration'
+        description: 'Describes the job configuration.'
+        required: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'jobType'
+            description: |
+              The type of the job.
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'jobTimeoutMs'
+            description: |
+              Job timeout in milliseconds. If this time limit is exceeded, BigQuery may attempt to terminate the job.
+          - !ruby/object:Api::Type::KeyValuePairs
+            name: 'labels'
+            description: |
+              The labels associated with this job. You can use these to organize and group your jobs.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'query'
+            description: 'Configures a query job.'
+            exactly_one_of:
+              - query
+              - load
+              - copy
+              - extract
+            properties:
+              - !ruby/object:Api::Type::String
+                name: 'query'
+                description: |
+                  SQL query text to execute. The useLegacySql field can be used to indicate whether the query uses legacy SQL or standard SQL.
+                required: true
+              - !ruby/object:Api::Type::NestedObject
+                name: 'destinationTable'
+                description: |
+                  Describes the table where the query results should be stored.
+                  This property must be set for large results that exceed the maximum response size.
+                  For queries that produce anonymous (cached) results, this field will be populated by BigQuery.
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'projectId'
+                    description: 'The ID of the project containing this table.'
+                    required: true
+                  - !ruby/object:Api::Type::String
+                    name: 'datasetId'
+                    description: 'The ID of the dataset containing this table.'
+                    required: true
+                  - !ruby/object:Api::Type::String
+                    name: 'tableId'
+                    description: 'The ID of the table.'
+                    required: true
+              - !ruby/object:Api::Type::Array
+                name: 'userDefinedFunctionResources'
+                description: |
+                  Describes user-defined function resources used in the query.
+                item_type: !ruby/object:Api::Type::NestedObject
+                  properties:
+                    - !ruby/object:Api::Type::String
+                      name: 'resourceUri'
+                      description: 'A code resource to load from a Google Cloud Storage URI (gs://bucket/path).'
+                      # TODO (mbang): exactly_one_of: resourceUri, inlineCode
```

Comment:
Can this TODO be removed / resolved? + below

Reply:
unfortunately, no, we can't add this to any children of an array yet